app/models/game.rb

```diff
@@ -1,2 +1,4 @@
 class Game < ActiveRecord::Base
+
+	belongs_to :user
```

Comment:
Pro tip!

In Ruby for indentation. We use "2 spaces" instead of tabs. You can set your editor up to do this automatically for you :).

Example: https://www.sublimetext.com/docs/2/indentation.html

More info: https://github.com/bbatsov/ruby-style-guide#spaces-indentation